pkg/hintrunner/zero/zerohint_memcpy.go

```diff
@@ -10,14 +10,27 @@ func createAllocSegmentHinter(resolver hintReferenceResolver) (hinter.Hinter, er
 	return &core.AllocSegment{Dst: hinter.ApCellRef(0)}, nil
 }
 
-func createVMEnterScopeHinter(resolver hintReferenceResolver) (hinter.Hinter, error) {
+func newMemcpyEnterScopeHinter(len hinter.ResOperander) hinter.Hinter {
```

Comment:
the `new ...` function for hints is supposed to end with `Hint`, not `Hinter`.

So `newMemcpyEnterScopeHint` would be the right name

if you look at uint256, usort, ec , keccak hints, you will see this is the convention. But indeed, in signatures hint , there is `newVerifyECDSASignatureHinter` , `newGetPointFromXHinter`, `newImportSecp256R1PHinter` and `newDivModSafeDivHinter` , it should be renamed with Hint at the end.

But functions starting with `create...` ends with `Hinter` , and here it is ok

Reply:
I see, thanks for the explanation. This should be fixed now.